src/app.ts

```diff
@@ -240,6 +240,10 @@ class TypescriptTextWriter implements ITypescriptTextWriter {
   public method(name: string, parameters: { parameter: string, type: string | TypescriptWriterCallback }[], returnType: string, singleLine = false) {
     this.writer.startIndentedLine(`${name}(`);
 
+    if (!singleLine) {
```

Comment:
Oops, just branched off the previous one so this is still here. Agree with your comment, let's get automatic prettier set up after 👍